ethereum-consensus/src/capella/block_processing.rs

```diff
@@ -37,10 +40,54 @@ pub fn process_bls_to_execution_change<
         BYTES_PER_LOGS_BLOOM,
         MAX_EXTRA_DATA_BYTES,
     >,
-    _signed_address_change: &mut SignedBlsToExecutionChange,
-    _context: &Context,
+    signed_address_change: &SignedBlsToExecutionChange,
+    context: &Context,
 ) -> Result<()> {
-    unimplemented!()
+    let address_change = &signed_address_change.message;
+    let signature = &signed_address_change.signature;
+
+    if address_change.validator_index >= state.validators.len() {
+        return Err(invalid_operation_error(InvalidOperation::ValidatorIndex(
+            address_change.validator_index,
+        )))
+    }
+
+    let withdrawal_credentials_prefix =
+        state.validators[address_change.validator_index].withdrawal_credentials[0];
+
+    if withdrawal_credentials_prefix != BLS_WITHDRAWAL_PREFIX {
+        return Err(invalid_operation_error(InvalidOperation::WithdrawalCredentialsPrefix(
+            state.validators[address_change.validator_index].withdrawal_credentials[0],
+        )))
+    }
+
+    let public_key: &BlsPublicKey = &address_change.from_bls_public_key;
+
```

Comment:
looks like we dropped the check that the remainder of the withdrawal credentials matches the hash of the `from_bls_pubkey`

Reply:
Fixed : https://github.com/ralexstokes/ethereum-consensus/pull/251/files#diff-a33e4ef5488fe0396a7616e8ded7baf22a8fec837a613e48edefee587a7c2666R70